clients/ts/signalr/tests/HubConnection.test.ts

```diff
@@ -53,11 +53,11 @@ describe("HubConnection", () => {
             const connection = new TestConnection();
             const hubConnection = createHubConnection(connection);
 
-            hubConnection.pingIntervalInMilliseconds = 150;
+            hubConnection.pingIntervalInMilliseconds = 5;
 
             try {
                 await hubConnection.start();
-                await delay(322);
+                await delay(32);
```

Comment:
Too small of a delay, on windows the minimum timer tick is 15 milliseconds, so this can be flaky. Just make it 200